insights/parsers/ironic_conf.py

```diff
@@ -0,0 +1,32 @@
+"""
+IronicConf - file ``/etc/ironic/ironic.conf``
+================================================
+This class provides parsing for the file ``/etc/ironic/ironic.conf``.
+Sample input data is in the format::
+    [DEFAULT]
+    auth_strategy=keystone
+    default_resource_class=baremetal
+    enabled_hardware_types=idrac,ilo,ipmi,redfish
+    enabled_bios_interfaces=no-bios
+    enabled_boot_interfaces=ilo-pxe,pxe
+    enabled_console_interfaces=ipmitool-socat,ilo,no-console
+
+    [agent]
+    deploy_logs_collect=always
+    deploy_logs_storage_backend=local
+    deploy_logs_local_path=/var/log/ironic/deploy/
+
+    [cinder]
+    auth_url=http://192.168.24.3:5000
+    project_domain_name=Default
+    project_name=service
+    user_domain_name=Default
+See the ``IniConfigFile`` class for examples.
+"""
+from .. import parser, IniConfigFile
+from insights.specs import Specs
+
+
+@parser(Specs.ironic_conf)
+class IronicConf(IniConfigFile):
+    pass
```

Comment:
@smalleni you need to add a single line docstring in this class so that it will be included in the generated documentation.